extrabees/src/main/java/binnie/extrabees/worldgen/WorldGenHiveNether.java

```diff
@@ -23,7 +23,7 @@ public WorldGenHiveNether(int rate) {
 	public boolean generate(final World world, final Random random, final BlockPos pos) {
 		final Biome biome = world.getBiome(pos);
 		if (!BiomeDictionary.hasType(biome, BiomeDictionary.Type.NETHER)) {
-			return true;
+			return false;
```

Comment:
I think this returned true to prevent the world generator from trying again in that dimension because it thinks the feature has been generated. It doesn't make a lot of difference though.

Reply:
I agree. But I made the changes, just to make the code more similar to the other hives.
https://github.com/ForestryMC/Binnie/blob/42435ea142bb28978625e461cd7b176af0d8979b/extrabees/src/main/java/binnie/extrabees/worldgen/WorldGenHiveMarble.java#L31-L33
https://github.com/ForestryMC/Binnie/blob/42435ea142bb28978625e461cd7b176af0d8979b/extrabees/src/main/java/binnie/extrabees/worldgen/WorldGenHiveWater.java#L24-L26